### PR TITLE
Change issuerKid to verificationMethodId in Issue VC blog post

### PR DESCRIPTION
**Pull request by bobbilee19**

### CODE REVIEW

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
+# Issue Verfiable Credentials
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:1]: Issue Credentials

writing 'Verifiable Credentials' is gonna get repetitive on the sidebar once we add more guides

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+This is a walk through of the Verifiable Credential (VC) issuance process from the
+perspective of the issuer. It is using a manual approach to issue verifiable
+credentials and is **not intended to be used in production applications**.
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:5]: is this still true?

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+## **Steps to Issue a Verifiable Credential**
+
+1. **[SSI Service Setup](#ssi-service-setup)**
+
+  - Clone & Run SSI Service
+  - Create Decentralized Identifier (DID) for Credential Issuer (Acme)
+  - Create or obtain DID for Credential Subject (Alice)
+  - Create Credential Schema
+
+2. **[Create Verifiable Credential](#create-verifiable-credential)**
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:27]: Create and Issue Verifiable Credential

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+
+```js
+{ "status":"OK" }
+```
+
+**b. Create Decentralized Identifier for Credential Issuer (Acme)**
+
+In the world of [Self-Sovereign Identity](/docs/glossary#self-sovereign-identity-ssi) (SSI), [Decentralized Identifiers](/docs/glossary#decentralized-identifier-did) (DIDs) are used to identify any subject (e.g., a person, organization, thing, data model, abstract entity, etc.). In this scenario, both Acme and Alice are represented by their DIDs in the VC we create in step two.
+
+:::info
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:73]: Can we remove lines 73-83 and instead just point them to the DID guide above?

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+
+The following response should be returned:
+
+```js
+{ "status":"OK" }
+```
+
+**b. Create Decentralized Identifier for Credential Issuer (Acme)**
+
+In the world of [Self-Sovereign Identity](/docs/glossary#self-sovereign-identity-ssi) (SSI), [Decentralized Identifiers](/docs/glossary#decentralized-identifier-did) (DIDs) are used to identify any subject (e.g., a person, organization, thing, data model, abstract entity, etc.). In this scenario, both Acme and Alice are represented by their DIDs in the VC we create in step two.
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:71]: change DID link to the guide on DIDs

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+  },
+  "credentialJwt": "eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6ejZNa2plUEc2VUJ..."
+}
+```
+
+:::note
+  The value for <b>credentialJwt</b> can be decoded using a tool like [jwt.io](https://jwt._io/).
+:::
+
+If Alice has an identity wallet she will be able to store the credential and use it however she wishes.
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:272]: is this the issuance of the VC or is another step required?

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+:::info
+  A [Credential Schema](https://www.w3.org/TR/vc-json-schema) is a document that defines the structure of a VC. It's based on the [json schema](https://json-schema.org) and shows which properties the issuer is going to use to create the VC.
+:::
+
+The `employedAt` property is a timestamp data type to define the date and time someone was employed at Acme.
+
+Let's create a Credential Schema for Alice's Employment Status VC:
+- Set `issuer` to Acme's DID.
+- Set `verificationMethodId` set to the KID of the issuer's private key to sign the schema.
+- Our `schema` object conforms to the Draft 2020-12 schema version, so set its `$schema` property to https://json-schema.org/draft/2020-12/schema.
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:180]: remove this as a link. it's downloading a file to people's computer and that's frowned upon, especially without warning

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/ssi/issue-verifiable-credentials.md [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+
+:::info
+  A [Credential Schema](https://www.w3.org/TR/vc-json-schema) is a document that defines the structure of a VC. It's based on the [json schema](https://json-schema.org) and shows which properties the issuer is going to use to create the VC.
+:::
+
+The `employedAt` property is a timestamp data type to define the date and time someone was employed at Acme.
+
+Let's create a Credential Schema for Alice's Employment Status VC:
+- Set `issuer` to Acme's DID.
+- Set `verificationMethodId` set to the KID of the issuer's private key to sign the schema.
```
Comment (angiejones) [inline site/docs/ssi/issue-verifiable-credentials.md:179]: is KID still the right word to use here?

File: site/docs/ssi/issue-verifiable-credentials.md (hunk 2 above)
Comment (bobbilee19) [reply site/docs/ssi/issue-verifiable-credentials.md:5]: this is what we had before, so has it changed to being prod or not?

Comment (bobbilee19) [reply site/docs/ssi/issue-verifiable-credentials.md:5]: Can we just more so say the SSI Service / TBD is in tech preview mode vs making it seem like VCs explicitly are not made for prod right now.

like:
`This is a walk through of the Verifiable Credential (VC) issuance process from the perspective of the issuer. It is using a manual approach to issue verifiable credentials and the SSI Service is still in tech preview and **not intended to be used in production applications**.`

File: site/docs/ssi/issue-verifiable-credentials.md (hunk 8 above)
Comment (bobbilee19) [reply site/docs/ssi/issue-verifiable-credentials.md:179]: we actually do not need this, it's not required actually after i did some digging, doing some digging here on SSI Service steps — in the schema creation step, we don’t need the `verificationMethodId` [see here](https://github.com/TBD54566975/ssi-service/blob/main/doc/howto/credential.md) and [here to confirm](https://github.com/TBD54566975/ssi-service/blob/main/pkg/service/schema/model.go#L30) It looks like it’s not required based on this, if it has that value then it validates it, if not it just returns nil) — so going to remove that for now to remove any confusion

Comment (bobbilee19) [reply site/docs/ssi/issue-verifiable-credentials.md:179]: so im going to remove this, it is still the KID field though yes

**Comment by angiejones** [comment]
@bobbilee19 there are broken links causing the build to fail

image

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/ssi/create-credentials.md [hunk 9]
```diff
@@ -0,0 +1,291 @@
...
+  - Clone & Run SSI Service
+  - Create Decentralized Identifier (DID) for Credential Issuer (Acme)
+  - Create or obtain DID for Credential Subject (Alice)
+  - Create Credential Schema
+
+**[2. Create a Verifiable Credential](#create-verifiable-credential)**
+
+<Divider type="slash" />
+
+## **1. SSI Service Setup**
```
Comment (angiejones) [inline site/docs/ssi/create-credentials.md:35]: isn't this redundant to what's in the Run SSI Service guide? Can we skip this section and simply point them to the guide on this?

**Review by angiejones — CHANGES_REQUESTED** [review]

**Comment by bobbilee19** [comment]
@angiejones I'm going to remove the SSI Service setup section and link to the SSI Service Setup Guide

File: site/docs/ssi/create-credentials.md (hunk 9 above)
Comment (bobbilee19) [reply site/docs/ssi/create-credentials.md:35]: yep just doing this now on the Create Did guide, so i just noted that below before i saw this comment!

File: site/docs/ssi/create-credentials.md [hunk 10]
```diff
@@ -0,0 +1,291 @@
...
+The VC is signed and packaged as a JWT in the credentialJwt property. If you're interested, you can decode the JWT using a tool such as [jwt.io](https://jwt._io/). If you were to `issue` or transmit the credential to a holder you would just send this JWT value.
+
+### Getting Credentials
+Once you've created multiple credentials, you can view all credentials by making a GET request to `/v1/credentials`. This endpoint also supports three query parameters: `issuer`, `schema`, and `subject` which can be used mutually exclusively.
+
+You can get a single credential by making a `GET` request to `/v1/credentials/{id}`.
+
+<Divider type="slash" />
+
+## Next Steps
```
[the remark this answers is not in the document]
Comment (bobbilee19) [reply site/docs/ssi/create-credentials.md:288]: do u want to remove next steps from all (like in the Create a DID doc im creating right now too?)

[the remark this answers is not in the document]
Comment (angiejones) [reply site/docs/ssi/create-credentials.md:288]: yes, I think it'll get redundant and doesn't really make sense to point to SSI Setup after they've already ready how to create, etc. Maybe we can come back after we have quite a few SSI guides and make more meaningful references in each versus the same ones in all.

**Review by angiejones — CHANGES_REQUESTED** [review]

**Review by angiejones — APPROVED** [review]

**Pull request merged**